code/modules/spells/spell_types/forcewall.dm

```diff
@@ -29,7 +29,6 @@
 /obj/effect/forcefield/wizard/Initialize(mapload, mob/summoner)
 	. = ..()
 	wizard = summoner
-	QDEL_IN(src, 300)
```

Comment:
?

Reply:
The other forcefields kept their delete timers in the forcefield themselves so I made it consistent.